### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/feature/notice.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#894] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/feature/notice.html

VuePress Theme Hope 允许你通过 [@vuepress/plugin-notice][notice] 添加公告。

https://theme-hope.vuejs.press/zh/guide/feature/notice.html

**Comment by Zephery** [comment]
公告的位置能自定义么？想放到右边导航栏下面。

**Comment by PhyQxx** [comment]
一个路径下支持多个公告吗，比如首页 path: '/'，想放多个公告
